prep-html-links/index.html

```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Home Page</title>
+</head>
+<body>
+  <h1>Home page</h1>
+  <a href="contact-us"> Click here to go to the contact page </a>
```

Comment:
Clicking this link should go to the `contact-us.html` file inside of the `contact-us` folder, so you'll need to change your `href` value to `"contact-us/contact-us.html"`